docs/how-to-guides/develop-custom-tool/specify-tool-structure.md

```diff
@@ -0,0 +1,23 @@
+# Tool Structure
```

Comment:
why do you make this part as a separate doc?

Reply:
Removed the tool structure doc from this pr, it can be added in another pr, thanks.